clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -73,9 +74,20 @@ public ProduceResponse(Map<TopicPartition, PartitionResponse> responses) {
      * @param throttleTime Time in milliseconds the response was throttled
      */
     public ProduceResponse(Map<TopicPartition, PartitionResponse> responses, int throttleTime) {
-        super(new Struct(CURRENT_SCHEMA));
+        this(responses, throttleTime, (int) ProtoUtils.latestVersion(ApiKeys.PRODUCE.id));
```

Comment:
The comment says this is the constructor for version 1, but the code uses the latest version.